examples/hello_world/test_driver/smoke_web_engine_test.dart

```diff
@@ -28,5 +31,20 @@ void main() {
     test('title is correct', () async {
       expect(await driver.getText(titleFinder), 'Hello, world!');
     });
+
+    test('enable accessibility', () async {
+      await driver.enableAccessibility();
+
+      await Future<void>.delayed(const Duration(seconds: 2));
```

Comment:
Is there any way we can remove this timeout? In general this kind of thing is a source of flakes, and thus violates our style guide (https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#never-check-if-a-port-is-available-before-using-it-never-add-timeouts-and-other-race-conditions).